library/src/main/java/com/otaliastudios/zoom/ZoomApi.kt

```diff
@@ -114,18 +114,30 @@ interface ZoomApi {
     annotation class ZoomType
 
     /**
-     * Defines the available transvormation types
+     * Defines the available transformation types
      */
     @Retention(AnnotationRetention.SOURCE)
     @IntDef(TRANSFORMATION_CENTER_INSIDE, TRANSFORMATION_CENTER_CROP, TRANSFORMATION_NONE)
     annotation class Transformation
 
     /**
-     * Defines the available smaller policies
+     * Defines the available alignments
      */
     @Retention(AnnotationRetention.SOURCE)
-    @IntDef(SMALLER_POLICY_CENTER, SMALLER_POLICY_FROM_TRANSFORMATION, SMALLER_POLICY_NONE)
-    annotation class SmallerPolicy
+    @IntDef(
+        com.otaliastudios.zoom.Alignment.BOTTOM,
+        com.otaliastudios.zoom.Alignment.CENTER_VERTICAL,
+        com.otaliastudios.zoom.Alignment.NONE_VERTICAL,
+        com.otaliastudios.zoom.Alignment.TOP,
+        com.otaliastudios.zoom.Alignment.LEFT,
+        com.otaliastudios.zoom.Alignment.CENTER_HORIZONTAL,
+        com.otaliastudios.zoom.Alignment.NONE_HORIZONTAL,
+        com.otaliastudios.zoom.Alignment.RIGHT,
+        com.otaliastudios.zoom.Alignment.CENTER,
+        com.otaliastudios.zoom.Alignment.NONE,
```

Comment:
I think the `Alignment` class could be imported, or is there another `Alignment` import already? Would look much cleaner imo.

Reply:
Oh I see it's because the annotation class is named Alignment too right?

Reply:
Yes! But this won't be a problem for the user as no-one will likely import the `Alignment` annotation.